src/extensions/registries/welcome-menu-registry.ts

```diff
@@ -1,7 +1,7 @@
 import { BaseRegistry } from "./base-registry";
 
 export interface WelcomeMenuRegistration {
-  title: string;
```

Comment:
Actually, given that this is just used a value. Shouldn't this be `React.ComponentLike`? @jakolehm ?

Reply:
I've been thinking about it as well. Decided agains it for the sake of simplicity and consistency in UI

Reply:
Maybe `React.ComponentType` or `React.ReactNode` (includes `string`)?